frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/FetchAllRemotesAction.java

```diff
@@ -19,6 +20,16 @@
 @CustomLog
 public class FetchAllRemotesAction extends BaseProjectDependentAction {
 
+  private static final long FETCH_ALL_UP_TO_DATE_TIMEOUT = 60 * 1000;
+
+  private static final java.util.Map<String, Long> LAST_FETCH_TIME_MILLIS_BY_REPOSITORY_NAME = new java.util.TreeMap<>();
```

Comment:
Hmm I'd say it should be `lowerCamelCase` as it's not a "constant" really

Reply:
Also, I'd use som concurrent impl since it's accessed not only from UI thread (see `run` below, multiple non-UI threads can execute that method concurrently) :/